tests/MongoDB.Driver.Tests/MongoClientSettingsTests.cs

```diff
@@ -398,12 +424,14 @@ public void TestFreezeInvalid()
         public void TestFromUrl()
         {
             // set everything to non default values to test that all settings are converted
+            // with the exception of tlsDisableCertificateRevocationCheck because setting that with tlsInsecure is
+            // not allowed in a connection string
             var connectionString =
                 "mongodb://user1:password1@somehost/?appname=app1;authSource=db;authMechanismProperties=CANONICALIZE_HOST_NAME:true;" +
                 "compressors=zlib,snappy;zlibCompressionLevel=9;connect=direct;connectTimeout=123;ipv6=true;heartbeatInterval=1m;heartbeatTimeout=2m;localThreshold=128;" +
                 "maxIdleTime=124;maxLifeTime=125;maxPoolSize=126;minPoolSize=127;readConcernLevel=majority;" +
                 "readPreference=secondary;readPreferenceTags=a:1,b:2;readPreferenceTags=c:3,d:4;retryReads=false;retryWrites=true;socketTimeout=129;" +
-                "serverSelectionTimeout=20s;tls=true;tlsInsecure=true;waitqueuesize=130;waitQueueTimeout=131;" +
+                "serverSelectionTimeout=20s;tls=true;sslVerifyCertificate=false;waitqueuesize=130;waitQueueTimeout=131;" +
```

Comment:
Why it was replaced? is this obsolete option really required?

Reply:
I think the only test you need is `tlsDisableCertificateRevocationCheck` which is the only scope of your ticket. 
I will rely on your and Robert's opinion here

Reply:
In this case we have 3 initial options controlling one setting. I would rather test it when I have an opportunity.